### ISSUE TITLE
Complies, won't link  errors with ld

### ISSUE THREAD

**Issue by jadepgit** [usnistgov/REFPROP-cmake#43]
Compiling in Linux.  Generates the proper fortran code.  Then is tries to create the library and pukes.  Here's a snip showing  some of the errors, and then I skip to the last error messages:

> /usr/bin/ld: CMakeFiles/refprop.dir/FORTRAN/DLLFILES/PASS_FTN.FOR.o: in function `refpropdll_':
> PASS_FTN.FOR:(.text+0x0): multiple definition of `refpropdll_'; CMakeFiles/refprop.dir/FORTRAN/PASS_FTN.FOR.o:PASS_FTN.FOR:(.text+0x0): first defined here
> /usr/bin/ld: CMakeFiles/refprop.dir/FORTRAN/DLLFILES/PASS_FTN.FOR.o: in function `refprop1dll_':
> PASS_FTN.FOR:(.text+0x5a0): multiple definition of `refprop1dll_'; CMakeFiles/refprop.dir/FORTRAN/PASS_FTN.FOR.o:PASS_FTN.FOR:(.text+0x5a0): first defined here
> /usr/bin/ld: CMakeFiles/refprop.dir/FORTRAN/DLLFILES/PASS_FTN.FOR.o: in function `refprop2dll_':
> PASS_FTN.FOR:(.text+0x9a0): multiple definition of `refprop2dll_'; CMakeFiles/refprop.dir/FORTRAN/PASS_FTN.FOR.o:PASS_FTN.FOR:(.text+0x9a0): first defined here
> 
> #NOTE:  Looks like I get this in every function.  Last error messages:
> 
> PASS_FTN.FOR:(.text+0x68b0): multiple definition of `phihmxdll_'; CMakeFiles/refprop.dir/FORTRAN/PASS_FTN.FOR.o:PASS_FTN.FOR:(.text+0x68b0): first defined here
> collect2: error: ld returned 1 exit status
> make[2]: *** [CMakeFiles/refprop.dir/build.make:313: librefprop.so] Error 1
> make[1]: *** [CMakeFiles/Makefile2:124: CMakeFiles/refprop.dir/all] Error 2
> make: *** [Makefile:103: all] Error 2

**Comment by jadepgit** [comment]
Hack fix:  I found 2 occurrences of PASS_FTN.FOR.o.  One in /FORTRAN and one in /FORTRAN/DLLFILES.  So I edited build.make, link.txt, and I believe depend.make and commented out all DLLFILES/PASS_FTN.FOR.o, and rm -r the DLL directory.  Then I reran cmake --build . , and it worked.  Result is now I have REFPROP.h and librefprop.so.

**Comment by ianhbell** [comment]
I think you have done something "creative" with your FORTRAN folder; PASS_FTN.FOR should only be in FORTRAN, or FORTRAN/DLLFILES, but not both.

Can you please post the full set of command line output from running your build after you make the build folder?

**Comment by jadepgit** [comment]
Hack fix:  I found 2 occurrences of PASS_FTN.FOR.o.  One in /FORTRAN and one in /FORTRAN/DLLFILES.  So I edited build.make, link.txt, and I believe depend.make and commented out all DLLFILES/PASS_FTN.FOR.o, and rm -r the DLL directory.  Then I reran cmake --build . , and it worked.  Result is now I have REFPROP.h and librefprop.so.

**Comment by jadepgit** [comment]
First, a big thank you for your support and creating this tool.

No need to run your test.  After your comment, I compared directory FORTRAN in my .wine folder and directory FORTRAN in my sandbox.  In .wine there is only one occurrence of PASS_FTN.FOR.FOR, and it is in the DLL directory.  Operator error on my part when I was troubleshooting some earlier compiler problems on my end.  (I'm a hobbyist).

And you might want to know that the front end Window's gui worked for me in wine.  I did the 32 bit trick just to be safe.  But I even ran a test case.

Thanks again for the great support.  This issue can be closed out.
